examples/components/MNIST/traininsilo/traininsilo.yaml

```diff
@@ -14,6 +14,16 @@ inputs:
   test_data:
     type: uri_folder
     description: the input testing data
+  silo_name:
```

Comment:
Could we instead use a non-FL name for this? for instance `metrics_prefix`. This way, it's still a generic component, but we use it in a way that fits in the FL scenario.

Reply:
oh makes sense.